includes/definitions/discovery/alcatel-omnipcx.yaml

```diff
@@ -0,0 +1,35 @@
+mib: ALCATEL-NMC-PROXY-AGENT-MIB:A4400-RTM-MIB:A4400-CPU-MIB
+modules:
+    sensors:
+        state:
+            data:
+                -
+                    oid: pbxRole
+                    value: pbxRole
+                    num_oid: .1.3.6.1.4.1.637.64.4400.1.4.
+                    descr: 'Pbx Role'
+                    index: 'pbxRole{{ $index }}'
+                    state_name: pbxRole
+                    states:
+                        - { descr: Main , graph: 0, value: 1 , generic: 0 }
+                        - { descr: Standby , graph: 0, value: 2 , generic: 0 }
+                        - { descr: Active PCS , graph: 0, value: 3 , generic: 0 }
+                        - { descr: Inactive PCS , graph: 0, value: 4 , generic: 0 }
+                        - { descr: Unknown , graph: 0, value: 0 , generic: 3 }
+                -
```

Comment:
You need states values fr all state sensors.

Reply:
I was afraid of this. But then, how could I have a sensor that just displays a "count" ? not a percentage, not a temperature, just a "count" ?

Reply:
I know, that's why I expected this ;) but the unit of the state sensor is # and it is exactly displayed like I would like to see ;) Except the title "State" of course ;)

Reply:
So, to recap, what should I do ? create a new type of sensor, named count, that would more or less share its code with state, except the state translation, and the event when the value changes ?

Reply:
@PipoCanaja yeah, best solution would be to create a new sensor type.

It would not share much with states.  Other than the display could be similar.

Reply:
All right, will have a look at that. Should I add that to the same PR or create a new one ?

Reply:
Did add it to this one, for now. Let's continue the discussion in the main conversation.